content/en/agent/amazon_ecs/data_collected.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Amazon ECS Data Collection
+kind: documentation
+further_reading:
+- link: "/agent/amazon_ecs/logs/"
+  tag: "Documentation"
+  text: "Collect your application logs"
+- link: "/agent/amazon_ecs/apm/"
+  tag: "Documentation"
+  text: "Collect your application traces"
+- link: "/agent/amazon_ecs/metrics/"
+  tag: "Documentation"
+  text: "Collect ECS metrics"
+---
+
+## Data collected
+
+### Metrics
+
+See [metadata.csv][1] for a list of metrics provided by this integration.
```

Comment:
this is actually the metrics from the AWS Web integration. @clamoriniere can you remind me where are the ECS + EC2 specific metrics documented (those that we are getting from the task metadata endpoint)?

Reply:
Updated this within [Kari's PR](https://github.com/DataDog/documentation/pull/8511)